reflex/reflex.py

```diff
@@ -255,6 +269,38 @@ def export(
         )
 
 
+db_cli = typer.Typer()
+
+
+@db_cli.command()
+def migrate():
+    """Create or update database schema based on app models or existing migration scripts."""
```

Comment:
Does this require that `makemigrations` is run first, and then will apply those?

Reply:
yes, `makemigrations` has to run first.

the `reflex db init` commit will initialize alembic, run `makemigrations` and then perform the migration. so after `init` the database is completely ready to go without further commands, until the schema changes.

Reply:
Can we move the docstring into the `help` kwarg - currently running `reflex db --help` cuts off the docstring for this method